syntax = "proto3";

package myrpc;

message RequestHeader {
    bytes service_name = 1; // The name of the service being called
    bytes method_name = 2;  // The name of the method being called
    uint32 args_size = 3;   // The size of the arguments being passed
}